scripts/flash/set-timestamp.js

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env node
+
+"use strict";
+
+var chalk = require("chalk");
+var displayTime = require("./display-time");
+
+function setTimestamp(augur, numberTicks, address, auth, callback) {
+  augur.api.TimeControlled.setTimestamp({
+    meta: auth,
+    tx: { to: address },
+    _timestamp: numberTicks,
+    onSent: function (result) {
+      console.log(chalk.yellow.dim("Sent Change Time"), chalk.yellow(JSON.stringify(result)));
+      console.log(chalk.yellow.dim("Waiting for reply ...."));
+    },
+    onSuccess: function (result) {
+      var timestamp = augur.api.Controller.getTimestamp();
```

Comment:
Suggest providing a callback to `augur.api.Controller.getTimestamp`, so that this request is not blocking (synchronous).